7_deploy_osh/roles/suse-osh-deploy/files/common-images-overrides.yml

```diff
@@ -27,8 +27,8 @@ images:
     heat_db_sync: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_osh_image_version }}"
     heat_engine: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_osh_image_version }}"
     heat_engine_cleaner: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_osh_image_version }}"
-    horizon: "{{ suse_osh_registry_location }}/openstackhelm/horizon:{{ suse_osh_image_version }}-clients"
-    horizon_db_sync: "{{ suse_osh_registry_location }}/openstackhelm/horizon:{{ suse_osh_image_version }}-clients"
+    horizon: "{{ suse_osh_registry_location }}/openstackhelm/horizon:queens-clients"
+    horizon_db_sync: "{{ suse_osh_registry_location }}/openstackhelm/horizon:queens-clients"
```

Comment:
Why is `{{ suse_osh_image_version }}` dropped here? (It would be correct to drop the `-clients` parts)

Reply:
Good catch: https://github.com/openstack/openstack-helm/blob/3cd4d0898ab517634d8c485d9e216150800f7828/horizon/values.yaml#L25